content/operate/rc/databases/rdi/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: Data Integration
+alwaysopen: false
+categories:
+- docs
+- operate
+- rc
+description: Use Redis Data Integration with Redis Cloud.
+hideListLinks: true
+weight: 99
+---
+
+Redis Cloud now supports [Redis Data Integration (RDI)]({{<relref "integrate/redis-data-integration">}}), a fast and simple way to bring your data into Redis from other types of primary databases.
+
+A relational database usually handles queries much more slowly than a Redis database. If your application uses a relational database and makes many more reads than writes (which is the typical case) then you can improve performance by using Redis as a cache to handle the read queries quickly. Redis Cloud uses [ingest]({{<relref "/integrate/redis-data-integration/">}}) to help you offload all read queries from the application database to Redis automatically.
+
+Having a data pipeline lets you have a cache that is always ready for queries. RDI Data pipelines ensure that any changes made to your primary database are captured in your Redis cache within a few seconds, preventing cache misses and stale data within the cache. 
```

Comment:
Minor, but "having lets you have" sounds a bit odd. Maybe "Using a data pipeline..." or "With a data pipeline you have.." or something like that?

Reply:
```suggestion
Using a data pipeline lets you have a cache that is always ready for queries. RDI Data pipelines ensure that any changes made to your primary database are captured in your Redis cache within a few seconds, preventing cache misses and stale data within the cache. 
```
```suggestion
Having a data pipeline lets you have a cache that is always ready for queries. RDI Data pipelines ensure that any changes made to your primary database are captured in your Redis cache within a few seconds, preventing cache misses and stale data within the cache. 
```